data/kestrel.txt

```diff
@@ -188,6 +189,7 @@ mission "Kestrel Available"
 		has "kestrel available"
 	destination Wayfarer
 	on offer
+		log "Got a message from Charles Atinoda at Tarazed. The 'Kestrel' I helped them design is available to buy."
```

Comment:
Single quotes around Kestrel to double quotes, backticks around the entire thing.